web/ww.ts

```diff
@@ -24,7 +24,7 @@ declare class Go {
 
 // The ICEServers JSON as exported from Pion capitalises field names, but
 // JS expects lowercase dictionary entries.
-type GoICEServers = [{ URLs: string[]; Username: string; Credential: string }];
+type GoICEServers = [{ urls: string[]; username: string; credential: string }];
```

Comment:
we can delete this now.

github won't let me comment on the correct lines, but lines 96 and 113 should now use `RTCIceServer[]` (built into typescript dom library) instead of `GoICEServers`